patches/chrome-android-java-src-org-chromium-chrome-browser-SwipeRefreshHandler.java.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java b/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
+index 9fa5c30608e83de37d682a7045c30f4643c7b1ff..ec0b3b9cf4428afb7e21c3036210a92c664fc967 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
+@@ -95,7 +95,7 @@ public class SwipeRefreshHandler
+      *
+      * @param tab The Tab where the swipe occurs.
+      */
+-    private SwipeRefreshHandler(Tab tab) {
++    public SwipeRefreshHandler(Tab tab) {
```

Comment:
I didn't found any better solution. If I would left it private, I couldn't do java asm patch.